src/app/CMakeLists.txt

```diff
@@ -159,7 +159,7 @@ if (WEBUI)
     target_link_libraries(qbt_app PRIVATE qbt_webui)
 endif()
 
-if (GUI)
+if (GUI AND (Qt5_VERSION VERSION_GREATER_EQUAL 5.14.0))
```

Comment:
I'm sorry, what's `qml_import_plugins` got to do with it?
 any case, if CMake is not able to build a workable application using statically linked Qt <5.14, it must fail at the configuration stage.

Reply:
it's not trivial to detect static build with CMake, I can give it a try, or can we just make it compile for now. It wouldn't have worked before anyway. Also, it's more of a Qt problem.

Reply:
#13456